src/FrameBase.re

```diff
@@ -47,3 +47,7 @@ external waitForXPath :
   (~xpath: string, ~options: selectorOptions=?, unit) =>
   Js.Promise.t(ElementHandle.t) =
   "";
+
+[@bs.send]
+external selectOneEval : (t, string, unit => unit) => Js.Promise.t('a) =
```

Comment:
So far we've been using the convention of having the object last so that things pipe nicely with `|>`. If you change this to `[@bs.send.pipe : t]` and adjust the test so it runs that would make it consistent with the rest of the bindings. I've got more work on the various eval bindings coming.